packages/lab/src/list/ListItem.css

```diff
@@ -1,15 +1,15 @@
 .uitkListItem {
   /* Color */
   --list-item-text-color: var(--uitk-selectable-foreground);
-  --list-item-background: var(--uitkListItem-background, var(--uitk-selectable-background));
+  --list-item-background: var(--uitkList-item-background, var(--uitk-selectable-background));
   --list-item-selected-focus-visible-outlineColor: var(--uitk-color-white);
   --list-item-text-color-active: var(--uitk-selectable-foreground-selected);
   --list-item-background-active: var(--uitk-selectable-background-selected);
   --list-item-alignItems: center;
 }
 
 .uitkListItem.uitkHighlighted {
-  --uitkCheckboxIcon-outlineColor: var(--uitk-selectable-borderColor-hover);
+  --uitkCheckbox-icon-outlineColor: var(--uitk-selectable-borderColor-hover);
```

Comment:
hummm... not for this PR, but changing the checkbox api in here doesn't seem right to me